test/mean-module.test.js

```diff
@@ -71,7 +73,11 @@ describe('Module Sub Generators Tests', function () {
       assert.file('modules/foo/client/services/');
       assert.file('modules/foo/client/views/');
       assert.file('modules/foo/tests/client/');
-      assert.file('modules/foo/server/config');
+      assert.file('modules/foo/server/config/');
+      assert.file('modules/foo/server/controllers/');
+      assert.file('modules/foo/server/models/');
+      assert.file('modules/foo/server/policies/');
+      assert.file('modules/foo/server/routes/');
       assert.file('modules/foo/tests/server/');
     });
```

Comment:
@mleanos The way prompts were set in it was wrong - they were set as an `Object`, while they should've been set as an `Array` of `String`s, because that's what CLI actually returns. Now it should be fine.